tests/test_symbolics.py

```diff
@@ -587,6 +590,50 @@ def test_minmax_precision(dtype, expected):
     assert np.all(f.data == 6.0)
 
 
+@pytest.mark.parametrize('dtype,expected', [
+    (np.float32, "powf"),
+    (np.float64, "pow"),
+])
+def test_pow_precision(dtype, expected):
+    grid = Grid(shape=(5, 5), dtype=dtype)
+
+    f = Function(name="f", grid=grid)
+    g = Function(name="g", grid=grid)
+
+    eqn = Eq(f, g**1.5)
+
+    op = Operator(eqn)
+
+    g.data[:] = 4.0
+
+    op.apply()
+
+    assert expected in str(op)
```

Comment:
If you check for `pow(` and `powf(` in the string, then you can also check that the other is not present

Reply:
Good point!